src/index.ts

```diff
@@ -853,7 +855,9 @@ export class SupraClient {
         chainId,
         maxGas,
         gasUnitPrice,
-        txExpiryTime
+        txExpiryTime === undefined
+          ? BigInt(Math.ceil(Date.now() / 1000) + 60 * 5)
```

Comment:
Can you please define constants for each of these numbers? That will make the code more readable and easier to maintain.